jsonify.js

```diff
@@ -2,7 +2,6 @@
  * Created by giova on 1/4/2017.
```

Comment:
What kind of license is your script being offered as? Is it GPL v2?

https://en.wikipedia.org/wiki/GNU_General_Public_License